README.md

```diff
@@ -27,11 +27,10 @@ in your IDE (like WebStorm).
 
 ## Dev Releases
```

Comment:
If you want to, feel free to add a section about Beta Releases here. Otherwise, I'll do it in a follow-up.

Reply:
I added a section about Beta releases similar to the Dev release section. Let me know what you think.

Reply:
Thanks!